fasttext/Rakefile

```diff
@@ -0,0 +1,109 @@
+UBUNTU_CODE_NAMES = "xenial,bionic,disco"
+UBUNTU_VERSIONS = "16.04,18.04,19.04"
+LAUNCHPAD_PPA = "groonga-ppa"
+
+apache_arrow_path = "../vendor/packages.red-data-tools.org/vendor/apache-arrow"
+require_relative "#{apache_arrow_path}/dev/tasks/linux-packages/package-task"
+
+ENV["APT_TARGETS"] ||= "debian-buster"
+ENV["CENTOS_VERSIONS"] ||= "7"
```

Comment:
`YUM_TARGETS`

Reply:
@kou could you update packages.red-data-tools.org/vendor/apache-arrow submodule?

Reply:
Done.

Reply:
Thanks!

Reply:
fixed.